api/v1alpha1/selfnoderemediationconfig_webhook_test.go

```diff
@@ -134,6 +155,7 @@ func testValidCR(validationType string) {
 	snrc.Spec.PeerRequestTimeout = &metav1.Duration{Duration: 30 * time.Second}
 	snrc.Spec.ApiCheckInterval = &metav1.Duration{Duration: 10*time.Second + 500*time.Millisecond}
 	snrc.Spec.PeerUpdateInterval = &metav1.Duration{Duration: 10 * time.Second}
+	snrc.Spec.CustomDsTolerations = []v1.Toleration{{Key: "validValue", Effect: v1.TaintEffectNoExecute}, {}, {Operator: v1.TolerationOpEqual, TolerationSeconds: pointer.Int64(-5)}, {Value: "SomeValidValue"}}
```

Comment:
this test should check if  the error contains all problems

Reply:
Hmm maybe I'm missing something IIUC these are valid values, the test check that there are no errors

Reply:
oh sorry, jumping too much around in the code. But maybe add it to testMultipleInvalidFields() above?

Reply:
Sure, sorry I've initial missed your latest respons -, for some reason this thread didn't automatically update in main conversation tab (I saw it once I switched to files tab).

Thanks for mentioning it again with the lgtm comment :+1: